pmm-app/src/pmm-qan/panel/provider/provider.tsx

```diff
@@ -1,8 +1,7 @@
-import React, { useCallback, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
The changes in this file don't seem to have any relationship to the scrollbars. Where do they belong?

Reply:
It is related to grafana upgrade